src/CanvasWrapper.js

```diff
@@ -603,9 +601,8 @@ export default class CanvasWrapper {
 
         // TODO: Better handle exceptions
         // Exception for nitro (draw nitro as NO2 instead of N+O-O)
-        if (charge === 1 && elementName === 'N' && attachedPseudoElement.hasOwnProperty('0O') &&
-            attachedPseudoElement.hasOwnProperty('0O-1')) {
-            attachedPseudoElement = { '0O': { element: 'O', count: 2, hydrogenCount: 0, previousElement: 'C', charge: '' } }
+        if (charge === 1 && elementName === 'N' && '0O' in attachedPseudoElement && '0O-1' in attachedPseudoElement) {

```

Comment:
This now allows inherited properties, but I can't imagine that change being a problem here.